randovania/generator/filler/retcon.py

```diff
@@ -367,7 +368,12 @@ def debug_print_weighted_locations(all_locations_weighted: WeightedLocations, pl
         print(f"[{player_states[owner.index].name}] {node_name} - {weight}")
 
 
-def should_be_starting_pickup(player: PlayerState, locations: WeightedLocations) -> bool:
+def should_be_starting_pickup(
+    player: PlayerState, locations: WeightedLocations, start_case: StartingItemBehavior
+) -> bool:
+    if start_case == StartingItemBehavior.CAN_NEVER_BE_STARTING:
+        return False
```

Comment:
you're going to need to add a new test that gets full coverage on this function. you'll want to mock the arguments